tests/_utils/get_leader

```diff
@@ -16,13 +23,13 @@ leader_name=""
 
 for ((k=0; k<10; k++)); do
     leader_name=$($binary -test.coverprofile="$TEST_DIR/cov.$TEST_NAME.dmctl.$ts.$pid.out" DEVEL -master-addr=$master_addr list-member --leader | grep name | gawk 'match($2,/"(.*)",/,a) {print a[1]}')
-    if [[ ! -z "$leader_name" ]]; then
+    if [ ! -z "$leader_name" ] || [ $leader_can_be_empty ]; then
```

Comment:
if `leader_can_be_empty`, can we just not call this `get_leader`?

Reply:
I think it's ok, just ignore get leader when evicting the last master in the integration test